Automattic/pocket-casts-android
# Improving the Wear OS podcasts page

## Description

This change improves the podcasts page in the following ways.

- Adds a podcast color gradient to the chip.
- Adds a page header. 

It also includes:

- Changing to the extra dark theme. This fixes the issue that you can see Pixel watch bezel if the background isn't black.
- Changing the theme typography to match the design.

## Testing Instructions
1. Tap "Podcasts"
2. ✅  Scroll down the list and verify the tint colours look fine.

## Screenshots
![Screenshot_20230524_203452](https://github.com/Automattic/pocket-casts-android/assets/308331/b3887e35-3e7f-465c-8fa3-2ff0dba5f495)
